apps/base-docs/tutorials/docs/5_shopify-storefront-commerce.md

```diff
@@ -0,0 +1,119 @@
+
+## Getting Started
+
+To comply with Shopify's requirements, the Coinbase Commerce account linked to Shopify must be a Coinbase Managed account.
+
+To create one, [sign up](https://beta.commerce.coinbase.com/sign_up) for a business account and select "Coinbase Commerce" as the account type. See this Coinbase Commerce Merchant [help article](https://help.coinbase.com/en/commerce/getting-started/merchant-guidelines-commerce-application) for additional information on how to sign your business up.
+
+### Link Coinbase Commerce to your Shopify store
+
+![shopify-install-commerce.gif](../../assets/images/shopify-storefront-commerce/shopify-install-commerce.gif)
+
+Once, you've created a Coinbase Commerce Managed account you can now add the plugin to your Shopify store.
+
+To add Coinbase Commerce as a payment method on your Shopify store, start by navigating to your admin page at `https://admin.shopify.com/store/<YOUR-STORE-NAME>`.
+Once there, click on Settings , located in the bottom left panel of the screen (or at `https://admin.shopify.com/store/<YOUR-STORE-NAME>settings/general`). Then select the “Payments” tab.
+Proceed by clicking “Add a payment method” and choose “Search by provider.” In the search field, type in “Coinbase” and select “Coinbase Commerce” from the results.
+Click "Install" and you should be redirected to Coinbase Commerce with aprompt log into your Coinbase account. Finally, click "Activate" to enable the Coinbase Commerce plugin.
+
+### Create a Storefront
+
+In this tutorial, we'll guide you through the steps to create a new Hydrogen storefront for your Shopify store. This will allow you to showcase the products you already have in your Shopify account. Make sure you've already created your products by following the [Shopify products guide](https://help.shopify.com/en/manual/online-sales-channels/shop/products-and-collections).
+
+To get started, we'll first clone a Shopify demo store using the Hydrogen framework. This will give us a quick setup to work with.
+
+```bash
+npm create @shopify/hydrogen@latest -- --quickstart
+```
+
+Once the demo store is cloned, navigate into the project directory and start the development server:
+
+```bash
+cd hydrogen-quickstart
+npx shopify hydrogen dev
+```
+
+Next, you'll need to link your Hydrogen project to your Shopify store. This step connects the demo storefront with your Shopify account, allowing you to display your products.
+
+```bash
+npx shopify hydrogen link
+```
+
+To ensure your storefront is properly configured, update the project environment variables. This pulls the necessary settings from your Shopify account into the Hydrogen project.
+
+```bash
+npx shopify hydrogen env pull
+```
+
+Now, let's verify that everything is set up correctly and your site is running. Start the development server again:
+
+```bash
+npx shopify hydrogen dev
```

Comment:
Has the command changed?  It told me:

│                                                                                                          │
│  Llllets Goooo is now linked                                                                             │
│                                                                                                          │
│  Next steps                                                                                              │
│    • Run `h2 dev` to start your local development server and start building

Reply:
That worked and I didn't have to pull .env

Reply:
Ahh, I see what's happening.  I'd tell them about mock.shop